src/vs/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -1046,6 +1091,12 @@ configurationRegistry.registerConfiguration({
 			default: true,
 			tags: ['notebookLayout']
 		},
+		[NotebookSetting.openOutputInPreviewEditor]: {
```

Comment:
We can align with other notebook output settings, for example `notebook.output.previewEditor`.